tests/test_metric_output.py

```diff
@@ -0,0 +1,39 @@
+import requests
+import yaml
+
+
+class TestMetricOutput:
+    def test_vcsa_data_comparison(self, setup_vcsa_url):
+        response = requests.get(setup_vcsa_url)
+        metrics = self.process_response_data(response)
+
+        with open('tests/vcsa_output_comparison.yaml', 'r') as fp:
+            comparison_data = yaml.safe_load(fp)
+
+        for key in comparison_data.keys():
+            difference = set(comparison_data[key]).difference(metrics)
+            assert len(difference) == 0
+
+    def test_vcenter_backend_down(self, request, setup_vcsa_url, setup_vcenter):
+        setup_vcenter.name = 'false'
```

Comment:
How about
```suggestion
        setup_vcenter.name = 'foobar-vcenter.whatever.domain'
```

Reply:
This is a good idea, it enhances readability.